routes/agreement-1/agreement-1.njk

```diff
@@ -2,6 +2,8 @@
 
 {% block content %}
 
+<a href={{ "documents/" + docxFilename }}>Dowload .docx</a>
```

Comment:
if we put the files in the `/tmp` dir then we wouldn't have to ever worry about deleting them ourselves - maybe worth doing?

Reply:
I was thinking about that - would I just create one in the public folder, like: `simplify-privacy-statements-V2/public/tmp`? And how does it work, does any folder with that name get automatically cleaned whenever the server restarts? Or does the folder have to be in a certain place?

Reply:
Ah, I meant the global `/tmp` folder that alpine uses.

Reply:
In that case I would have to add a route for that - I'll give it a try in a future PR